Lib/collections/__init__.py

```diff
@@ -47,7 +47,7 @@ def __getattr__(name):
         obj = getattr(_collections_abc, name)
         import warnings
         warnings.warn("Using or importing the ABCs from 'collections' instead "
-                      "of from 'collections.abc' is deprecated since Python 3.3,"
+                      "of from 'collections.abc' is deprecated since Python 3.3, "
                       "and in 3.9 it will stop working",
```

Comment:
This might not be necessary, but it looks like it's also missing a period at the end. 

```suggestion
                       "and in 3.9 it will stop working.",
```

Reply:
FWIW looks like most warning messages (`grep -R -F -A2 'warnings.warn(' Lib`) don't have a trailing dot.

Reply:
Oh my mistake then, feel free to disregard.